packages/edit-site/src/components/global-styles/style.scss

```diff
@@ -56,7 +56,8 @@
 	padding: 0 $grid-unit-20;
 }
 
-.edit-site-block-types-search {
+.edit-site-block-types-search,
+.edit-site-block-types-item-list {
```

Comment:
`.edit-site-block-types-item-list` doesn't need a bottom margin. So the code should be like this:

```scss
.edit-site-block-types-search {
	margin-bottom: $grid-unit-10;
	padding: 0 $grid-unit-20;
}
.edit-site-block-types-item-list {
	padding: 0 $grid-unit-20;
}
```